lib/lightning_web/live/credential_live/credential_form_component.ex

```diff
@@ -528,19 +528,32 @@ defmodule LightningWeb.CredentialLive.CredentialFormComponent do
   end
 
   defp get_type_options(schemas_path) do
+    manifest =
+      case get_manifest_path() |> File.read() do
```

Comment:
@elias-ba , this is cool, but still always evaluates to `nil` in the built image. (my earlier attempt didn't work either.) is that because it's being evaluated before `install adaptor icons` happens?